docs/lib/storage/fragments/ios/download.md

```diff
@@ -96,7 +96,26 @@ receiveValue: {
 
 </amplify-block-switcher>
 
-### Generate a download URL
+## Cancel, Pause, Resume
+
+After you call `downloadData` or `downloadFile` as above, your containing class retains a reference to the operation that is actually performing the download.
```

Comment:
```suggestion
Calls to `downloadData` or `downloadFile` return a reference to the operation that is actually performing the download.
```

Reply:
Take this suggestion with a grain of salt, but it's up to the calling class whether to retain the reference, so it may be simpler just to say that the reference is returned.